File: DesafiosPython/URI1010.py
PRODUTO_1 = input().split(' ')
PRODUTO_2 = input().split(' ')
cod1 = int(PRODUTO_1[0])
qtd1 = int(PRODUTO_1[1])
preco1 = float(PRODUTO_1[2])
cod2 = int(PRODUTO_2[0])
qtd2 = int(PRODUTO_2[1])
preco2 = float(PRODUTO_2[2])
TOTAL = (qtd1*preco1)+(qtd2*preco2)

print(f'VALOR A PAGAR: R$ {TOTAL:.2f}')

File: DesafiosJava/URI1010.java
import java.util.Scanner;

public class URI1010 {
    public static void main(String[] args) {
        String prod;
        int qtd1, qtd2;
        double val_unid1, val_unid2, total;

        Scanner entrada = new Scanner(System.in);
        prod = entrada.nextLine();

        String[] valores = prod.split(" ");
        qtd1 = Integer.parseInt(valores[1]);
        val_unid1 = Double.parseDouble(valores[2]);

        prod = entrada.nextLine();
        valores = prod.split(" ");
        qtd2 = Integer.parseInt(valores[1]);
        val_unid2 = Double.parseDouble(valores[2]);

        total = (qtd1*val_unid1)+(qtd2*val_unid2);

        System.out.printf("VALOR A PAGAR: R$ %.2f\n", total);
        entrada.close();
    }
}
